devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -212,6 +212,8 @@ public class DevMojo extends AbstractMojo {
     @Parameter(defaultValue = "${debug}")
     private String debug;
 
+    @Parameter(defaultValue = "${openLoom}")
+    private boolean openLoom;
```

Comment:
Maybe this flag should have a more generic name? It doesn't appear to be Loom specific